instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2ClientTest.groovy

```diff
@@ -73,6 +80,106 @@ abstract class AbstractAws2ClientTest extends InstrumentationSpecification {
 
   abstract void configureSdkClient(SdkClientBuilder builder)
 
+  def "send DynamoDB #operation request with builder #builder.class.getName() mocked response"() {
+    setup:
+    configureSdkClient(builder)
+    def client = builder
+      .endpointOverride(server.address)
+      .region(Region.AP_NORTHEAST_1)
+      .credentialsProvider(CREDENTIALS_PROVIDER)
+      .build()
+    responseBody.set("")
+    def response = call.call(client)
+
+    if (response instanceof Future) {
+      response = response.get()
+    }
+
+    expect:
+    response != null
+    response.class.simpleName.startsWith(operation)
+    assertDynamoDbRequest(service, operation, path, method, requestId)
+
+    where:
+    [service, operation, method, path, requestId, builder, call] << dynamoDbRequestDataTable()
+  }
+
+  def "send DynamoDB #operation async request with builder #builder.class.getName() mocked response"() {
```

Comment:
I don't see how this test differs from the previous one

Reply:
Good catch, forgot to propagate non-statement changes from the "original" PR.